ramls/link.raml

```diff
@@ -85,7 +85,7 @@ traits:
           orderBy:
             displayName: Field to order by
             type: string
```

Comment:
Can we use there enum instead of a string?

Reply:
will do in the next PR related to note https://issues.folio.org/browse/MODNOTES-146